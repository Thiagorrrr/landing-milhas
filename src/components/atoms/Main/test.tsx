import { screen, render } from '@testing-library/react'

import { Main } from './index'

describe('Main test', () => {
  it('should render the heading test', () => {
    render(<Main />)

    expect(
      screen.getByRole('heading', { name: /Sua loja Pet/i })
    ).toBeInTheDocument()
  })
  it('should render snapshot', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toMatchSnapshot()
  })

  it('Should render style color correctly', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toHaveStyle({
      'background-color': '#06092b'
    })
  })
})
