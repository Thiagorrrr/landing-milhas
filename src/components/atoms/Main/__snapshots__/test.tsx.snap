// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main test should render snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: #06092b;
  color: #fff;
  width: 100%;
  height: 100%;
  text-align: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 3rem;
}

.c1 {
  width: 25rem;
  margin-bottom: 2rem;
}

.c2 {
  font-size: 2.5rem;
}

.c3 {
  font-size: 2rem;
  font-weight: 400;
}

<main
  class="c0"
>
  <img
    alt="Imagem de uma logo"
    class="c1"
    src="img/logo.svg"
  />
  <h1
    class="c2"
  >
    Sua loja Pet
  </h1>
  <h2
    class="c3"
  >
    Sua nova loja pet 
  </h2>
  <img
    alt="Imagem de uma ilustração"
    class="c1"
    src="img/hero-illustration.svg"
  />
</main>
`;
